src/components/v2v3/shared/FundingCycleConfigurationDrawers/NftDrawer/NftUpload.tsx

```diff
@@ -91,8 +85,24 @@ export default function NftUpload({ form }: { form: FormInstance }) {
         beforeUpload={beforeUpload}
         customRequest={async req => {
           setUploading(true)
-          const res = await pinFileToIpfs(req.file)
-          setValue(res.IpfsHash)
+          setPercent(0)
+          try {
+            const val = await pinFileToIpfs({
+              ...req,
+              onProgress: percent => {
+                if (percent) {
+                  setPercent(percent)
+                }
+              },
+            })
+            setValue(val.IpfsHash)
+          } catch (e) {
+            console.error('Error occurred while uploading', e)
+            // eslint-disable-next-line @typescript-eslint/no-explicit-any
+            req.onError?.(null as any)
```

Comment:
could this be `unknown` instead, to avoid `any`?

Reply:
![image](https://user-images.githubusercontent.com/104132113/212215788-909ae707-23a9-4f89-bf92-5fbd1f016919.png)
:(